### PR TITLE
Add Yarna Armoured Coat

### PR DESCRIPTION
**Pull request by Frostbitey**
## About the PR
Added the Yarna's Armoured Coat!

## Why / Balance
As a personal item for Yarna-Crimson, Aka. OverwatchVoice
The Yarna's Amoured Coat has all armor statistics as the Warden's Armoured Winter Coat

## Media
Preview:
image
image
image
image

**Changelog**

:cl:
- add: Added Yarna's Armoured Coat!!

### CODE REVIEW

File: Resources/Prototypes/_Umbra/Entities/Clothing/OuterClothing/coats.yml [hunk 1]
```diff
@@ -10,3 +10,25 @@
...
+    sprite: _Umbra/Clothing/OuterClothing/Coats/yarnaarmouredcoat.rsi
+  - type: Clothing
+    sprite: _Umbra/Clothing/OuterClothing/Coats/yarnaarmouredcoat.rsi
+  - type: Armor
+    modifiers:
+      coefficients:
+        Blunt: 0.70
+        Slash: 0.70
+        Piercing: 0.70
+        Heat: 0.70
```
Comment (TsjipTsjip) [inline Resources/Prototypes/_Umbra/Entities/Clothing/OuterClothing/coats.yml:32]: Not allowed in a personal item (inherit from a warden's coat instead).

File: Resources/Prototypes/_Umbra/Entities/Clothing/OuterClothing/coats.yml [hunk 2]
```diff
@@ -10,3 +10,25 @@
...
+  - type: ExplosionResistance
+    damageCoefficient: 0.90
```
Comment (TsjipTsjip) [inline Resources/Prototypes/_Umbra/Entities/Clothing/OuterClothing/coats.yml:34]: Not allowed in a personal item (inherit from a warden's coat instead).

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_Umbra/Entities/Clothing/OuterClothing/coats.yml [hunk 3]
```diff
@@ -10,3 +10,25 @@
...
   - type: Clothing
     sprite: _Umbra/Clothing/OuterClothing/Coats/coatmoonlit.rsi
+
+# Player: OverwatchVoice - Character: Yarna Crimson
+- type: entity
+  parent: ClothingOuterStorageBase
+  id: PersonalItemClothingOuterYarnaArmourCoat
+  name: Yarna's armoured winter coat
+  description: A elegant winter coat made with high-quality fibers, featuring two golden shoulder guards, there is a small label that reads "Yarna-Crimson".
+  suffix: PersonalItem
```
[the remark this answers is not in the document]
Comment (Frostbitey) [reply Resources/Prototypes/_Umbra/Entities/Clothing/OuterClothing/coats.yml]: Problems Fixed

File: Resources/Prototypes/_Umbra/Entities/Clothing/OuterClothing/coats.yml (hunk 2 above)
Comment (Frostbitey) [reply Resources/Prototypes/_Umbra/Entities/Clothing/OuterClothing/coats.yml]: Problems FIxed

**Review by TsjipTsjip — APPROVED** [review]
This only conveys a GM approval for the concept of the item, and not of the item actually in the game, as I have not been able to test it in-game yet. YML looks good tho.

**Review by TsjipTsjip — APPROVED** [review]
Works after local testing. Good to merge.

**Pull request merged**